webcomponents/tool/src/main/frontend/packages/sakai-conversations/src/SakaiConversationsTagManager.js

```diff
@@ -24,10 +24,25 @@ export class SakaiConversationsTagManager extends SakaiElement {
   _createTags() {
 
     const field = this.querySelector("#tag-creation-field");
+    const rawValue = field?.value || "";
+
+    const parsedTags = [];
+    if (rawValue.trim().length > 0) {
+      const regex = /"([^"]*)"|([^,]+)/g;
+      let match;
+      while ((match = regex.exec(rawValue)) !== null) {
+        // match[1] is the content of a quoted string (e.g., "tag, with comma" -> tag, with comma)
+        // match[2] is an unquoted segment (e.g., simpletag)
+        // Prioritize quoted content, then unquoted, then trim.
+        const tag = match[1] ? match[1].trim() : (match[2] ? match[2].trim() : "");
+        if (tag.length > 0) {
+          parsedTags.push(tag);
+        }
+      }
+    }
```

Comment:
TODO: move all of this to server-side processing using a standard CSV library for parsing